op-e2e/system_adminrpc_test.go

```diff
@@ -172,6 +177,54 @@ func TestLoadSequencerStateOnStarted_Started(t *testing.T) {
 	assertPersistedSequencerState(t, stateFile, node.StateStarted)
 }
 
+func TestPostUnsafePayload(t *testing.T) {
+	InitParallel(t)
+	ctx := context.Background()
+
+	cfg := DefaultSystemConfig(t)
+	cfg.Nodes["verifier"].RPC.EnableAdmin = true
+
+	sys, err := cfg.Start(t)
+	require.NoError(t, err)
+	err = sys.BatchSubmitter.Stop(ctx)
```

Comment:
I think you need to avoid starting the batcher at all, otherwise it may submit a batch before you shut it down:
```suggestion
	cfg.DisableBatcher = true
	sys, err := cfg.Start(t)
	require.NoError(t, err)
```